### PR TITLE
Add docs for mpm deploy command

### PR DESCRIPTION
**Pull request by 0xpause**
The binary tool `mpm` added a subcommand `deploy` , [PR](https://github.com/starcoinorg/starcoin/pull/3495).
These are the related docs.

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/98-reference/01-cli/02-mpm.md [hunk 1]
```diff
@@ -204,19 +205,19 @@ mpm release [OPTIONS]
   - 如果功能用，将打印额外的诊断信息。
 
 
-### `mpm-integration-test` 详述
+### `mpm deploy` 详述
```
Comment (geometryolife) [inline i18n/zh/docusaurus-plugin-content-docs/current/98-reference/01-cli/02-mpm.md:208]: 新特性把 `mom-integration-test` 移除了？
@0xpause @jolestar

Comment (0xpause) [reply i18n/zh/docusaurus-plugin-content-docs/current/98-reference/01-cli/02-mpm.md:208]: 没有移除mpm integration test 命令。应该是误改的，我检查一下。

**Comment by jolestar** [comment]
可以等新版本 release 再合并进去。

File: i18n/zh/docusaurus-plugin-content-docs/current/98-reference/01-cli/02-mpm.md (hunk 1 above)
Comment (0xpause) [reply i18n/zh/docusaurus-plugin-content-docs/current/98-reference/01-cli/02-mpm.md:208]: fixed

**Review by geometryolife — COMMENTED** [review]
LGTM

**Review by jolestar — APPROVED** [review]

**Comment by jolestar** [comment]
release https://github.com/starcoinorg/starcoin/releases/tag/v1.11.12

**Pull request merged**